testing/file_test/test_file.cpp

```diff
@@ -93,9 +96,47 @@ struct SplitState {
   int file_index = 0;
 };
 
+static auto ExtractFilePathFromUri(llvm::StringRef uri) -> llvm::StringRef {
+  static constexpr llvm::StringRef FilePrefix = "file:/";
+  if (uri.starts_with(FilePrefix)) {
+    return uri.drop_front(FilePrefix.size());
+  }
+  return uri;
+}
+
+static auto AutoFillDidOpenParams(llvm::json::Object* params,
+                                  llvm::ArrayRef<TestFile::Split> splits)
+    -> ErrorOr<Success> {
+  auto* text_document = params->getObject("textDocument");
+  if (text_document == nullptr) {
+    return Success{};
+  }
+
+  auto attr_it = text_document->find("text");
+  if (attr_it == text_document->end() || attr_it->second != "AUTOFILL") {
```

Comment:
minor but i suggest a different wording here
```suggestion
  if (attr_it == text_document->end() || attr_it->second != "FROM_FILE_SPLIT") {
```

Reply:
Done